src/app/views/App.tsx

```diff
@@ -351,13 +352,13 @@ class App extends Component<IAppProps, IAppState> {
                 {mobileScreen && appTitleDisplayOnMobileScreen(
                   stackTokens,
                   classes,
+                  minimised,
                   this.toggleSidebar)}
 
                 {!mobileScreen && appTitleDisplayOnFullScreen(
                   classes,
                   minimised,
-                  this.toggleSidebar
-                )}
+                  this.toggleSidebar)}
```

Comment:
Is there a reason for this? I would personally argue the code was cleaner/easier to read before

Reply:
i'm not exactly sure what you are referring to 😅

Reply:
This commit section takes a 2 line and just removes a newline so that it becomes a one-liner